distributed/tests/test_cancelled_state.py

```diff
@@ -944,6 +953,31 @@ def test_workerstate_resumed_fetch_to_executing(ws_with_running_task):
     assert ws.tasks["x"].state == prev_state
 
 
+def test_workerstate_resumed_fetch_to_executing(ws_with_running_task):
```

Comment:
```suggestion
def test_workerstate_resumed_fetch_to_executing(ws_with_running_task):
    """See test_resumed_cancelled_handle_compute for end-to-end version"""
```

Reply:
These low level tests are pretty valuable but they _should_ always have an accompanying high level test. This is the case here and I recommend adding this as a comment.
This test encodes an assumption about the behavior of the scheduler which may not be true forever